pagopa/io-app
# feat: [IOPLT-394] Bump Xcode version used to build the app

## Short description
This PR bumps xcode version on CI to be compliant with app store.
Uses arm runners to improve execution time.

## List of changes proposed in this pull request
- Changed the runner and setup on `release.yml` workflow

## How to test
Manually run a release workflow changing the workflow trigger or just check the previous run [here](https://github.com/pagopa/io-app/actions/runs/8362091374?pr=5611)
